### ISSUE TITLE
React: Can't resolve '../../charting_library/charting_library.min' 

### ISSUE THREAD

**Issue by bfriel** [tradingview/charting-library-examples#192]
**Checklist**

- [x] This issue is **NOT** related to the charting library itself

**Describe the bug**

I am unable to compile the [React Integration Example](https://github.com/tradingview/charting-library-examples/tree/master/react-javascript). After following the four steps under the "How to Start" section, I see the following error:

```
./src/components/TVChartContainer/index.jsx
Module not found: Can't resolve '../../charting_library/charting_library.min' 
```

**To Reproduce**

1. Clone the repo
2. Go to `react-javascript` folder
3. Run `npm install`
4. Copy `charting_library` from https://github.com/tradingview/charting_library/ (Release v17.021) to `/public` and `/src` folder
5. Copy `datafeeds` folder from https://github.com/tradingview/charting_library/ (Release v17.021) to `/public`
6. Run `npm start`
7. See error

**Expected behavior**

The example would compile

**Screenshots**

N/A

**Additional context**

N/A

**Comment by eugene-korobko** [comment]
Hello!

It looks like the sampe is looking for minimized build of the library.
Could you please check your `./src/charting_library` folder if it has 'charting_library.js' or 'charting_library.min.js'
If it has charting_library.js, just change ./src/components/TVChartContainer/index.jsx file with replacing 'charting_library.min.js' with 'charting_library.js'

**Comment by bhp-work** [comment]
I am also facing the same issue! I replaced the file as suggested by eugene-korobko , but still error is there.

**Comment by eugene-korobko** [comment]
Hello!

Could you please check what charting_library* files are in the ./src/charting_library folder?

**Comment by bhp-work** [comment]
![crp](https://user-images.githubusercontent.com/70111150/98344438-405d1980-2039-11eb-8a4f-3e4831cdb510.png)

oops! I guess I have put charting_library and datafeed in 'public' folder and not in 'src' folder. 
I cloned the charting library again and now I have the charting_library/charting_library.min.js file.  But the error is still there.

@eugene-korobko  shall I move the charting_library inside src folder?

**Comment by eugene-korobko** [comment]
Hello!

According to the readme.md file, 

1. Copy `charting_library` folder from https://github.com/tradingview/charting_library/ to `/public` and to `/src` folders. The earliest supported version of the Charting Library is 1.12. If you get 404 then you need to [request an access to this repository](https://www.tradingview.com/HTML5-stock-forex-bitcoin-charting-library/).
1. Copy `datafeeds` folder from https://github.com/tradingview/charting_library/ to `/public`.

**Comment by bhp-work** [comment]
Hi,

Now I have copied chartling_library to /public **AND** to src folder. 
Module error is resolved, but its throwing error on page 

I have charting_library.min.js file in the src/charting_library.


![crp](https://user-images.githubusercontent.com/70111150/98348665-0c84f280-203f-11eb-8969-ea6e706944c2.png)

**Comment by bhp-work** [comment]
Hi,

Now I have copied charting_library to /public **AND** to src folder.
Module error is resolved, but its throwing error on page

I have **charting_library.min.js** file in the src/charting_library.

![crp](https://user-images.githubusercontent.com/70111150/98352900-9edbc500-2044-11eb-94d4-67b0ec2de2ce.png)

**Comment by bhp-work** [comment]
Hello,

My error is resolved. Below paths worked for me....

public/index.html => 
 `<script src="charting_library/datafeeds/udf/dist/polyfills.js"></script>
  <script src="datafeeds/udf/dist/bundle.js"></script>`

src/components/TVChartContainer/index.jsx =>
`import {widget} from '../../charting_library/charting_library.min';`

Also I compiled the project using below commands. But this compilation is not really necessary. 
PS D:\"ProjectFolderPath"> cd D:\"ProjectFolderPath"\public\datafeeds\udf
npm install -g rollup
npm install --save-dev rollup-plugin-buble
npm i rollup-plugin-uglify
npm install --save-dev rollup-plugin-node-resolve
npm run compile
npm run bundle-js
PS D:\"ProjectFolderPath"> npm start  

Happy coding!

**Comment by bfriel** [comment]
Hi @eugene-korobko ,

I was able to compile by switching the link from `charting_library.min` to `charting_library.js`. I should note, however, that this also threw a `'define' is not defined` error outlined [here](https://github.com/tradingview/charting-library-examples/issues/36). I was able to resolve that by referencing [this comment](https://github.com/tradingview/charting-library-examples/issues/36#issuecomment-411023104).

I have no further issues, but I hope that someone else will find this if they also come across this error.
